lms/templates/course.html

```diff
@@ -2,29 +2,27 @@
 <%!
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
-from courseware.courses import get_course_about_section
-from openedx.core.lib.courses import course_image_url
 %>
 <%page args="course" />
-<article class="course" id="${course.id | h}" role="region" aria-label="${get_course_about_section(request, course, 'title')}">
+<article class="course" id="${course.id | h}" role="region" aria-label="${course.display_name_with_default}">
   <a href="${reverse('about_course', args=[course.id.to_deprecated_string()])}">
     <header class="course-image">
       <div class="cover-image">
-        <img src="${course_image_url(course)}" alt="${get_course_about_section(request, course, 'title')} ${course.display_number_with_default}" />
+        <img src="${course.course_image_url}" alt="${course.display_name_with_default} ${course.display_number_with_default}" />
         <div class="learn-more" aria-hidden=true>${_("LEARN MORE")}</div>
       </div>
     </header>
     <div class="course-info" aria-hidden="true">
       <h2 class="course-name">
-        <span class="course-organization">${get_course_about_section(request, course, 'university')}</span>
+        <span class="course-organization">${course.display_org_with_default}</span>
         <span class="course-code">${course.display_number_with_default}</span>
-        <span class="course-title">${get_course_about_section(request, course, 'title')}</span>
+        <span class="course-title">${course.display_name_with_default}</span>
       </h2>
       <div class="course-date" aria-hidden="true">${_("Starts")}: ${course.start_datetime_text()}</div>
     </div>
     <div class="sr">
       <ul>
-      <li>${get_course_about_section(request, course, 'university')}</li>
+      <li>${course.display_org_with_default}</li>
```

Comment:
Again, display_name_with_default is the only one that comes escaped (and it shouldn't), so this should be escaped.

FYI: Someone is working on a commit to temporarily rename display_name_with_default to display_name_w_default_escaped, which is deprecated.  If that were complete we could actually html escape the real method we want (display_name_with_default).  See https://github.com/agaylard/edx-platform/commit/f8c48b2c31c5d22aa5048f357fe0b8e408fa6e41